ui/scripts/movie.js

```diff
@@ -109,6 +123,31 @@ export default class Movie {
         return conteiner;
     }
 
+    // expects countries object that has name and flag fields, class  
+    getFlags(counries, name) {
+        let flags = "";
+
+        if (counries.length !== 0) {
```

Comment:
it's usually better to avoid indentation levels by returning earlier. E.g. here you could:

```
if (countries.length == 0) {
  return "";
}
<proceed handling the list>
```